### ISSUE TITLE
`go get` Functionality

### ISSUE THREAD

**Issue by timfallmk** [vapor-ware/synse-cli#114]
`go get` can be strict about what it expects to find in a given repo. Right now `go get github.com/vapor-ware/synse-cli` (once 2.0 lands) results in the following error:

```shell
can't load package: package github.com/vapor-ware/synse-cli: no Go files in /go/src/github.com/vapor-ware/synse-cli
```

This is because `go get` expects to find top level `.go` files and build only those. It does not descend the directory tree.

There are two ways of fixing this. Idiomatically, most go projects that have a `cmd/...` structure use submodules to let you checkout a specific command directory (e.g. `go get github.com/goland/dep/cmd/dep`). I don't want to use submodules. Like, at all.

The second way, which I think is less bad, is to include a top level `.go` shell file that simply redirects the builder to the right place. This should be fairly easy to add and would make it easier for people to just `go get`.

**Edit**: The third "way", of course, is just to leave it as is, and document how you should build. `go get -d` will suppress the error as well.

**Comment by edaniszewski** [comment]
Im not sure I have an opinion one way or another, so whatever you think is best/easiest from a usage perspective is good by me!

**Comment by timfallmk** [comment]
I think I'm way over thinking this. This _should_ ™ work:

`go get github.com/vapor-ware/synse-cli/cmd/`

..._as soon as_ that's merged into `master`

(`go` imports don't allow versioning, and the `gopkg.in` convenience doesn't work well with self-referencing packages)
